src/adUnit/VastAdUnit.js

```diff
@@ -202,6 +208,34 @@ class VastAdUnit extends VideoAdUnit {
     this[_protected].finish();
   }
 
+  /**
+   * Returns the duration of the ad Creative or 0 if there is no creative.
+   *
+   * @returns {number} - the duration of the ad unit.
+   */
+  duration () {
+    if (!this.isStarted()) {
+      return 0;
+    }
+    const {videoElement} = this.videoAdContainer;
+
+    return videoElement.duration;
+  }
+
+  /**
+   * Returns the current time of the ad Creative or 0 if there is no creative.
+   *
+   * @returns {number} - the current time of the ad unit.
+   */
+  currentTime () {
+    if (!this.isStarted()) {
+      return 0;
+    }
+    const {videoElement} = this.videoAdContainer;
+
+    return videoElement.currentTime;
```

Comment:
same, no need for am intermediate constant

Reply:
👍